src/main/java/org/mitre/inferno/App.java

```diff
@@ -1,6 +1,9 @@
 package org.mitre.inferno;
 
 import org.mitre.inferno.rest.ValidatorEndpoint;
+import org.mitre.inferno.utils.SparkUtils;
```

Comment:
```
> Task :checkstyleMain
[ant:checkstyle] [WARN] /home/src/main/java/org/mitre/inferno/utils/RequestLogFactory.java:4: Wrong lexicographical order for 'org.eclipse.jetty.server.CustomRequestLog' import. Should be before 'org.eclipse.jetty.server.Slf4jRequestLogWriter'. [CustomImportOrder]
[ant:checkstyle] [WARN] /home/src/main/java/org/mitre/inferno/utils/RequestLogFactory.java:7: Wrong lexicographical order for 'java.io.IOException' import. Should be before 'org.slf4j.Logger'. [CustomImportOrder]
[ant:checkstyle] [WARN] /home/src/main/java/org/mitre/inferno/utils/SparkUtils.java:4: Wrong lexicographical order for 'org.eclipse.jetty.server.CustomRequestLog' import. Should be before 'org.slf4j.Logger'. [CustomImportOrder]
```

Reply:
Good catch. I did run CheckstyleMain, and thought I had fixed this...apparently not.